### DISCUSSION TITLE
D1 mini and 2 switches via MQTT

### DISCUSSION THREAD

**Discussion by speedAmaster** [arendst/Tasmota#12680] [Q&A] [answered]
I am total newbie to Tasmota and am failing to handle an es easy job like:

* I have 2 reed-switches connected to D7(GPIO13) and D8 (GPIO15) configured as switch1 and switch2 (in tasmota-moduleConfig-Generic18) - same in templateConfig
* I did switchmode1 1 and switchmode2 1
* console shoes me for both switches: 
         stat/tasmota_7C98C5/RESULT = {"POWER":"ON"}
         stat/tasmota_7C98C5/POWER = ON
* I see no SENSOR_Switch1 or SENSOR_SWITCH2 change!

what do I need to do to simply fire a MQQT status for each switch?

**Comment by sfromis** [comment, marked as answer]
For MQTT messages for switch inputs, without relay action, use `SetOption114` or `SwitchMode 15`

**Reply by speedAmaster** [reply]
awesome! thanks sfromis!
I did
		SetOption114 1
		SwitchMode1 1
		SwitchMode2 1

and it works now.
THANKS a LOT!
